__tests__/app.test.js

```diff
@@ -74,39 +74,114 @@ describe("app", () => {
         });
     });
     describe("/api/articles/:article_id", () => {
-      test("200: responds with 200 status and an article object with the correct id", () => {
-        return request(app)
-          .get("/api/articles/2")
-          .then((response) => {
-            const { body } = response;
-            expect(body).toHaveProperty("author", expect.any(String));
-            expect(body).toHaveProperty("title", expect.any(String));
-            expect(body).toHaveProperty("article_id", expect.any(Number));
-            expect(body).toHaveProperty("body", expect.any(String));
-            expect(body).toHaveProperty("topic", expect.any(String));
-            expect(body).toHaveProperty("created_at", expect.any(String));
-            expect(body).toHaveProperty("votes", expect.any(Number));
-            expect(body).toHaveProperty("article_img_url", expect.any(String));
-          });
-      });
-      test("400: responds with 400 status and a message when sent bad request", () => {
-        return request(app)
-          .get("/api/articles/hello")
-          .expect(400)
-          .then(({ body }) => {
-            expect(body).toHaveProperty("msg", "bad request");
-          });
+      describe("GET /api/articles/:article_id", () => {
+        test("200: responds with 200 status and an article object with the correct id", () => {
+          return request(app)
+            .get("/api/articles/2")
+            .then((response) => {
+              const { body } = response;
+              expect(body).toHaveProperty("author", expect.any(String));
+              expect(body).toHaveProperty("title", expect.any(String));
+              expect(body).toHaveProperty("article_id", expect.any(Number));
```

Comment:
I would at the minimum check that the article_id is 2! Gives you proof that you have the correct articles back and not just any article